src/realm/object-store/sync/generic_network_transport.hpp

```diff
@@ -205,10 +206,17 @@ struct Request {
     bool uses_refresh_token = false;
 };
 
+typedef enum { success, failure } ResponseStatus;
```

Comment:
This should be an enum class and not use `typedef` (that's strictly a C thing).

Reply:
Thanks, fixed.